Student request

Lleiro opened the pull request:
IUT2 informatique in Grenoble France

ghost commented:
Pull request

[file.zip](https://github.com/HaiDesu/swot/files/5619404/file.zip)

villat committed:
Added INSR college

emman-infinity committed:
Add files via upload

WijayaSaputra-867 committed:
Add SMK TI DWIGUNA domain

philipto committed:
Merge pull request #39856 from WijayaSaputra-867/master

Pull request accepted.

philipto committed:
Renaming file according the repository rules

philipto committed:
Merge pull request #39855 from emman-infinity/master

Pull request accepted.

uzzalkar-mgi committed:
Add AKPI Bangladesh domain

philipto committed:
Merge pull request #39852 from villat/master

Pull request accepted.

philipto committed:
1. The Islamia University of Bahawalpur
2. Reorganized stoplist.txt

philipto committed:
Merge remote-tracking branch 'origin/master'

sandeshdandin committed:
Request to add Sanjay Ghodawat University

gummy-smiley committed:
add Rajinibon School

gummy-smiley committed:
Merge pull request #1 from gummy-smiley/rajinibon-school

add Rajinibon School

philipto committed:
Merge pull request #39863 from gummy-smiley/master

Pull request accepted.

philipto committed:
Merge pull request #39858 from uzzalkar-mgi/master

Pull request accepted.

NeuTroNBZh committed:
Add files via upload

Rayo-Dev10 committed:
Add files via upload

Añadir la Institución Universitaria Oriente de Caldas

philipto committed:
Cleanup

philipto committed:
Merge remote-tracking branch 'origin/master'

Feuerball500 committed:
Add files via upload

Please add my School.

philipto committed:
Merge pull request #39872 from Feuerball500/master

Pull request accepted.

philipto committed:
Merge pull request #39871 from Rayo-Dev10/master

Pull request accepted.

philipto committed:
Merge pull request #39870 from NeuTroNBZh/master

Pull request accepted.

shreesthapit1 committed:
Add lacm.edu.np

Add lacm.edu.np

philipto committed:
Merge pull request #39882 from shreesthapit1/master

Pull request accepted.

marcospestana committed:
Add files via upload

marcospestana committed:
Delete file.zip

marcospestana committed:
Add files via upload

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Cleanup

philipto committed:
Fraud

philipto committed:
Merge pull request #39888 from marcospestana/master

Pull request accepted.

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Fraud

philipto committed:
Fraud

james-d-elliott committed:
Add St Peter's College to domains list

marsalsoren1660-stack committed:
Registration for College

riku-mito committed:
Add files via upload

munna119 committed:
add-iom-domain

philipto committed:
Merge pull request #39905 from munna119/master

Pull request accepted.

philipto committed:
Merge pull request #39903 from riku-mito/master

Pull request accepted.

philipto committed:
Merge pull request #39902 from marsalsoren1660-stack/master

Pull request accepted.

philipto committed:
Update by request

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Update by request, Ecole professionnelle commerciale du canton du Jura

asyrafi-dev committed:
Add mhs.uinbukittinggi.ac.id

philipto committed:
Merge pull request #39928 from asyrafi-dev/master

Pull request accepted.

jayarajvamadevan committed:
Add files via upload

philipto committed:
Illinois Central College

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Illinois Central College

philipto committed:
Merge pull request #39930 from jayarajvamadevan/master

Pull request accepted.

yShinjiNwu committed:
Add files via upload

philipto committed:
Merge pull request #39935 from yShinjiNwu/master

Pull request accepted.

mmasinas committed:
Create utoronto.txt

Microwame committed:
Added Agrupamento de Escolas de Pombal

philipto committed:
Merge pull request #39940 from Microwame/aep

Pull request accepted.

philipto committed:
Merge pull request #39939 from mmasinas/master

Pull request accepted.

burnsallprimaryschool committed:
Request to add

We would like to add Burnsall Voluntary Aided Primary School in the UK.

The school's official government link is: https://get-information-schools.service.gov.uk/Establishments/Establishment/Details/121622

Please verify and approve this; we need to develop a school early childhood education system.

philipto committed:
Merge pull request #39943 from burnsallprimaryschool/master

Pull request accepted.

ChobitsChii committed:
Add rgs.bayern

Add Richard-Glimpel-Schule (rgs.bayern)

philipto committed:
Merge pull request #39960 from ChobitsChii/ChobitsChii-patch-1

Pull request accepted.

sonic-onkaringale committed:
Add NK Orchid College of Engineering & Technology, Solapur (https://orchidengg.ac.in/)

kreatzzz committed:
Add Sri Guru Gobind Singh College of Commerce

philipto committed:
Merge pull request #39968 from kreatzzz/add-sggscc-domain

Pull request accepted.

philipto committed:
Merge pull request #39965 from sonic-onkaringale/master

Pull request accepted.

goodderr committed:
Add files via upload

goodderr committed:
Add SID – Scuola Italiana Design (scuolaitalianadesign.com)

philipto committed:
Merge pull request #39969 from goodderr/master

Pull request accepted.

nezumi788 committed:
Add files via upload

https://iatk.edu.ua/

nezumi788 committed:
Merge pull request #1 from nezumi788/nezumi788-patch-1

Add files via upload

gopal-ssasc committed:
Official Website: https://www.sankaracollege.edu.in/

philipto committed:
Merge pull request #39979 from gopal-ssasc/master

Pull request accepted.

philipto committed:
Merge pull request #39976 from nezumi788/master

Pull request accepted.

kthweminhan committed:
Add KCG information in kcg.txt

philipto committed:
Merge pull request #39981 from kthweminhan/patch-1

Pull request accepted.

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Merge pull request #37619 from Sarvar-1993/master

Add Tashkent Institute of Chemical Technology

philipto committed:
Merge pull request #37648 from nikhilagarwal03/master

Add smec.ac.in Domain

TAHPAPANGKORN committed:
Add psp school domain

philipto committed:
Merge pull request #39994 from TAHPAPANGKORN/add-psp-school

Pull request accepted.

mihokote committed:
Add generated files

nikjp2021 committed:
Add Shizuoka University to shizuoka.txt

### Verification Information
- **Official Website:** https://www.shizuoka.ac.jp
- **IT/Computer Science Course Evidence:** https://www.inf.shizuoka.ac.jp/english/ (Faculty of Informatics)
- **Domain Email Proof:** The domain `shizuoka.ac.jp` is the standard academic suffix issued to all active students and faculty members at Shizuoka University.

philipto committed:
Merge pull request #40006 from nikjp2021/patch-3

Pull request accepted.

philipto committed:
Merge pull request #40004 from mihokote/add-generated-files

Pull request accepted.

philipto committed:
University of Hafr Albatin
Protecting from frauds

philipto committed:
Merge remote-tracking branch 'origin/master'

20260527 committed:
Add files via upload

panglongshibufei committed:
Add files via upload

Add Hunan College of Information (hniu.cn)

geloquan committed:
Add STI West Negros University to domains

name: Marco Angelo Quanico
email: quanico.388873@sti.wnu.edu.ph

philipto committed:
Renaming file according the repository rules

philipto committed:
Merge pull request #40022 from geloquan/master

Pull request accepted.

philipto committed:
Merge pull request #40020 from panglongshibufei/master

Pull request accepted.

philipto committed:
Merge pull request #40019 from 20260527/master

Pull request accepted.

manojvignesh committed:
Add files via upload

manojvignesh committed:
Remove duplicate line in psnacet.txt

philipto committed:
Merge pull request #40026 from manojvignesh/manojvignesh-patch-1

Pull request accepted.

IamArayel committed:
Merge branch 'master' into add-expernet-dot-re

zyond26 committed:
Create cmcu

zyond26 committed:
Add CMC University domain

Commit:
Add Faculdades ITES - Fundação Educacional de Taquaritinga

karthik1289 committed:
Adding Foundation School India

Adding Foundation School India

philipto committed:
Merge pull request #40030 from karthik1289/master

Pull request accepted.

philipto committed:
Merge pull request #40029 from SartoriaLab/add-ites

Pull request accepted.

sugitanaoki-agr committed:
Create a new branch for this commit and start a pull request

Please add Utsunomiya University.

philipto committed:
Merge pull request #37454 from saliba-siaucom/master

Atualizar arquivos

philipto committed:
Merge pull request #37463 from tainm230690-collab/master

Add onbav.edu.vn for Banking Academy of Vietnam (Distance Learning)

philipto committed:
Cleanup

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Merge pull request #37604 from YadneshTeli/master

Add nmitd.edu.in and deccansociety.org domains

philipto committed:
Merge pull request #37636 from iubi-bia/master

Adding London College of Contemporary Arts (LCCA) domains

philipto committed:
Merge pull request #37655 from Devsed-official/master

Add islec.edu.in domain for ISL Engineering College

philipto committed:
Merge pull request #37651 from ficrammanifur/add-uca-domain

Add University Cendekia Abditama domain uca.ac.id

philipto committed:
Merge pull request #37661 from ndo-fer/master

Add all student subdomains for PENS (*.student.pens.ac.id)

philipto committed:
Merge pull request #37678 from aitdcode/master

Add files via upload

philipto committed:
Merge pull request #37685 from toufikswar/master

Adding Geneva Business School

philipto committed:
Merge pull request #37695 from emewahyudi/master

Add SMAN 8 PEKANBARU to school list

philipto committed:
Merge pull request #37715 from kattapik/master

Add swry.txt with school name

philipto committed:
Merge pull request #37829 from MiguelGitSTI/master

Added STI Ortigas-Cainta

amityadav-72 committed:
Create prpotepatilengg.txt

philipto committed:
Merge pull request #37730 from hayuabdisa007-tech/master

Create ftveti.txt

philipto committed:
Rename lib/domains/ac/geidai/fm.txt to lib/domains/jp/ac/geidai/fm.txt

justlx committed:
Add lesna.edu.pl

mulalicd committed:
Add files via upload

aikkay-git committed:
Delete lib/domains/nl/cvo.txt

aikkay-git committed:
Merge pull request #37972 from Jesse-Voo/add-melanchthon-cvo

Add Melanchthon Scholengroep and CVO (ChristenVO) domains

philipto committed:
Rename lib/domains/ac/poltekba.txt to lib/domains/id/ac/poltekba.txt

philipto committed:
Merge pull request #39897 from james-d-elliott/feat-add-stpeters-vic-edu-au

feat: add stpeters.vic.edu.au

PhuDev2009 committed:
Add files via upload

philipto committed:
Rename lib/domains/ac/rgukt/edu.txt to lib/domains/in/ac/rgukt/edu.txt

philipto committed:
Merge pull request #38069 from jourdashe/master

Add STI College San Fernando

philipto committed:
Merge pull request #40049 from PhuDev2009/master

Pull request accepted.

philipto committed:
Merge pull request #40047 from justlx/master

Pull request accepted.

philipto committed:
Merge pull request #40046 from amityadav-72/master

Pull request accepted.

philipto committed:
Merge pull request #40038 from sugitanaoki-agr/master

Pull request accepted.

philipto committed:
Delete lib/domains/vn/edu/cmcu

philipto committed:
Merge pull request #40028 from zyond26/master

JetBrains Products for Learning

philipto committed:
Merge pull request #38165 from vigneshwari-14052005/master

Add Sri Parasakthi College to the domain list

philipto committed:
Merge pull request #39186 from godishalagoutham/master

Pull request accepted.

philipto committed:
Merge pull request #38052 from m2azura125/patch-1

Pull request accepted.

philipto committed:
Merge pull request #37918 from canvassoft/master

Pull request accepted.

philipto committed:
Merge pull request #40054 from mulalicd/master

Pull request accepted.

philipto committed:
Merge pull request #38120 from ahmedgalal007/master

Register the domain of Coptic College for Girls School

philipto committed:
Merge pull request #38150 from eduardonovaroma/master

Add faculdadenovaroma.txt file

philipto committed:
Rename lib/domains/ac/sac.txt to lib/domains/in/ac/sac.txt

philipto committed:
Merge pull request #38261 from wheresmurad/master

Add azmiu.txt with university name

philipto committed:
Merge pull request #38245 from Epiczhul/master

Create vvpbk.txt

philipto committed:
Rename lib/domains/ac/shizuoka.txt to lib/domains/jp/ac/shizuoka.txt

philipto committed:
Merge pull request #38632 from bongbub/master

안양대학교 anyang university

philipto committed:
Merge pull request #38549 from IamArayel/add-expernet-dot-re

Add domain for Expernet IT AND DIGITAL MARKETING CAMPUS

philipto committed:
Merge pull request #38481 from ZittaLuthfiyyah/master

Add Universidad Bethesda

kiangssk committed:
Add i-CATS University College domain entry

philipto committed:
Merge pull request #40064 from kiangssk/add-icats-domain

Pull request accepted.

philipto committed:
Merge pull request #39796 from Mdsaleh99/master

Pull request accepted.

philipto committed:
Merge pull request #39753 from nikjp2021/patch-1

Pull request accepted.

philipto committed:
Merge pull request #38010 from rawhuul/master

Add ssi.txt with Symbiosis Statistical Institute name

philipto committed:
Merge pull request #38018 from olivierensea/master

Add files via upload

philipto committed:
Rename in.txt to .txt

philipto committed:
Rename lib/domains/in/usharama/.txt to lib/domains/in/usharama.txt

philipto committed:
Rename lib/domains/ac/sanjayghodawatuniversity.txt to lib/domains/in/ac/sanjayghodawatuniversity.txt

philipto committed:
Merge pull request #38289 from MarkMallari16/master

Add STI College Las Pinas

philipto committed:
Merge pull request #38276 from JonathanKPS/master

Add regionvarmland.se & fhsregionvarmland.se domains

philipto committed:
Merge pull request #38172 from aadithyaa9/master

Add Shiv Nadar University Chennai

aikkay-git committed:
Merge pull request #39774 from KhaledZarrug/master

College of Electronic Technology

aikkay-git committed:
Merge pull request #39767 from namooplol/master

Create sakolraj.txt

aikkay-git committed:
Merge pull request #39732 from CronusRO2/master

Add mpasa.utm.md

philipto committed:
Cleanup

philipto committed:
Merge remote-tracking branch 'origin/master'

philipto committed:
Fraud

philipto committed:
Merge pull request #38057 from T0ksito/master

Create uii.txt

aikkay-git committed:
Merge pull request #39719 from oherranmo/master

Add Institut de Sils

Zews21 committed:
Add files via upload

philipto committed:
Merge pull request #40070 from Zews21/master

Pull request accepted.

philipto committed:
Merge pull request #39906 from sandeshdandin-sgu/main

Pull request accepted.

philipto committed:
Merge pull request #38022 from pavankalyanchittala/patch-2

Pull request accepted.

ihecker committed:
edu

philipto committed:
Merge pull request #40076 from ihecker/master

Pull request accepted.

prathamamritkar committed:
Add AISSMS College of Engineering to domains

080bct001abishekkhatri committed:
Create jec.txt

Add Janakpur Engineering College (jec.edu.np), Nepal

prathamamritkar committed:
Add AISSMS College of Engineering to text file

prathamamritkar committed:
Delete lib/domains/com/aissmscoe

philipto committed:
Merge pull request #40088 from prathamamritkar/master

Pull request accepted.

philipto committed:
Merge pull request #40087 from 080bct001abishekkhatri/master

Pull request accepted.

jitendradara12 committed:
Add JUET 

for the domain juet.ac.in

btx-systems committed:
Add Glendowie College to gdc.txt

philipto committed:
Merge pull request #40107 from btx-systems/master

Pull request accepted.

philipto committed:
Merge pull request #40105 from jitendradara12/master

Pull request accepted.

4mt21ec020 committed:
Add files via upload

philipto committed:
Merge pull request #40111 from 4mt21ec020/master

Pull request accepted.

edgza committed:
Add files via upload

akhil12121 committed:
Create cmpcollege.txt

Official website: https://cmpcollege.ac.in
Students email include @cmpcollege.ac.in and @liveclass.cmpcollege.ac.in

jithukrishnaju-tech committed:
sctce domain

philipto committed:
Merge pull request #38773 from JustinCoding19/master

rvhs.txt

zimi17 committed:
Add stiedwimulya.txt with school name of Sekolah Tinggi Ilmu Ekonomi Dwimulya

Arsh-Pathan committed:
Add files via upload

philipto committed:
Merge pull request #40125 from Arsh-Pathan/master

Pull request accepted.

shunpeinomura committed:
Add new file s.txt with university name

philipto committed:
Merge pull request #40122 from zimi17/master

Pull request accepted.

philipto committed:
Merge pull request #40115 from kotlinknight/sctce-domain

Pull request accepted.

philipto committed:
Merge pull request #40113 from edgza/master

Pull request accepted.

choco586 committed:
Add school metadata for Saint George University

imamfirmansyah committed:
Add SMA Negeri 6 Depok to school list

philipto committed:
Merge pull request #40135 from imamfirmansyah/patch-1

Pull request accepted.

philipto committed:
Merge pull request #40133 from choco586/add-my-university

Pull request accepted.

philipto committed:
Merge pull request #40131 from shunpeinomura/master

Pull request accepted.

j38600 committed:
adding escola profissional profitecla

https://www.profitecla.pt/a-profitecla/cursos/programador-informatica/

AmeyPacharkar1896 committed:
Add pvppcoe.txt with college information

akhil12121 committed:
Merge branch 'JetBrains:master' into master

philipto committed:
Merge pull request #40144 from akhil12121/master

Pull request accepted.

philipto committed:
Merge pull request #40142 from AmeyPacharkar1896/master

Pull request accepted.

philipto committed:
Merge pull request #40140 from j38600/master

Pull request accepted.

aikkay-git committed:
Hanoi University of Civil Engineering

aikkay-git committed:
Merge remote-tracking branch 'origin/master'

aahan0511 committed:
Add Anand Niketan Satellite Domain

dileepbalineni committed:
Add svcolleges.txt with college name entries

This domain belongs to Sri Venkateswara College of Engineering, Tirupati.

Official website:
https://svcolleges.edu.in/

The institution offers long-term academic programs in engineering and related fields.

Student email addresses are issued under this domain:
example: dileep.b@svcolleges.edu.in

Proof attached:

College website link : https://svcolleges.edu.in/

Screenshot of student email / portal showing the domain

philipto committed:
Merge pull request #40155 from dileepbalineni/patch-4

Pull request accepted.

philipto committed:
Merge pull request #40154 from aahan0511/master

Pull request accepted.

Odnacum committed:
Add Universidade Jean Piaget de Moçambique (unipiaget.ac.mz)

mnhr committed:
Add new file s.txt with university name

ilogdapl committed:
Add files via upload

TeamSquidTempura committed:
Add graded.txt for The American School of São Paulo

philipto committed:
Merge pull request #40166 from TeamSquidTempura/master

Pull request accepted.

philipto committed:
Merge pull request #40163 from ilogdapl/master

Pull request accepted.

philipto committed:
Merge pull request #40162 from muelabo/master

Pull request accepted.

philipto committed:
Merge pull request #40159 from Odnacum/add-unipiaget-mz

Pull request accepted.

sisayz committed:
Add MicroLink Information Technology College file

The official website of our college: https://microlinkitcollege.edu.et/

philipto committed:
Merge pull request #40170 from sisayz/master

Pull request accepted.

philipto committed:
Fraud

philipto committed:
Merge remote-tracking branch 'origin/master'

aikkay-git committed:
National Autonomous University of Mexico (UNAM) - Faculty of Sciences.

aikkay-git committed:
Merge remote-tracking branch 'origin/master'

vinay01467 committed:
Update iimk.txt

enkaku202606 committed:
Add Okayama Shoka University (base domain is osu.ac.jp)

philipto committed:
Merge pull request #40181 from enkaku202606/add-osu-mse

Pull request accepted.

vinay01467 committed:
Merge branch 'JetBrains:master' into master

vinay01467 committed:
Delete iimk.txt

eamalindu committed:
Add Metropolitan College information

Institution Name
Metropolitan College, Sri Lanka

Website
https://www.metropolitancollege.lk/

Email Domain
metropolitancollege.edu.lk

Evidence
Metropolitan College is a higher education institution in Sri Lanka offering diploma, degree and postgraduate programmes.

Official website:
https://www.metropolitancollege.lk/

philipto committed:
Merge pull request #40184 from eamalindu/master

Pull request accepted.

philipto committed:
Merge pull request #40183 from vinay01467/master

Pull request accepted.

RolasUST committed:
Add Universitas Katolik Santo Thomas (ust.ac.id)

Institution Name:
Universitas Katolik Santo Thomas

Website:
https://www.ust.ac.id

Domain:
ust.ac.id

The university provides official email addresses for students and staff using this domain.

Evidence:
https://www.ust.ac.id

philipto committed:
Merge pull request #40188 from RolasUST/patch-3

Pull request accepted.

ax-7 committed:
Create enstp.txt

added "École Nationale Supérieure des Travaux Publics"

dokterkepin committed:
Update isu.txt

philipto committed:
Merge pull request #40189 from ax-7/master

Pull request accepted.

Commit:
Add eulji

philipto committed:
Merge pull request #40192 from mina1130/master

Pull request accepted.

philipto committed:
Merge pull request #40190 from dokterkepin/master

Pull request accepted.
